### PR TITLE
Adds auto scroll up when element is selected

### PR DESCRIPTION
**Pull request by ericanaughton**
:tea: 

ready to go

### CODE REVIEW

File: js/scripts.js [hunk 1]
```diff
@@ -37,10 +37,20 @@ var app = {
...
 
+      app.setScroll(url);
+
     });
 
   },
 
+  setScroll: function(url) {
+    if(url === "welcome.html") {
+      $('html, body').animate( {scrollTop:0} );
```
Comment (rylnd) [inline js/scripts.js]: Can't these selectors just be `$("body")` instead of `$("html, body")`?

Comment (rylnd) [inline js/scripts.js]: For consistency with the rest of the function calls in this code, please remove the spaces around the parens:

``` js
$('html, body').animate({scrollTop:0});
```

**Comment by rylnd** [comment]
:+1: when comments are addressed.

**Comment by rylnd** [comment]
:+1:

**Comment by bigtiger** [comment]
:+1:

**Pull request merged**